### PR TITLE
Add Scripts for time calculation

### PR DESCRIPTION
**Pull request by Mitra98t**
### time-between-dates

Given two dates (the first is optional, the default value is today) the script returns the time between them in different unit of measure.

The result will be in milliseconds, hours, days and years.

### time-calculator

Given a date (optional, the default value is today) the script allows you to add or subtract an amount of time specifying the unit of measure choosing from Seconds, Minutes, Hours, Days, Weeks, Years.

The result will be the resulting date and time as well as the unix time.

## Type of change

- [x] New script command

### time-between-dates

TimeBetweenDates

### time-calculator

TimeCalculator

## Dependencies / Requirements

Install Nodejs

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/time-between-dates.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// @raycast.schemaVersion 1
+// @raycast.title Time Between Dates
+// @raycast.mode fullOutput
+
+// Optional parameters:
+// @raycast.icon 🕒
+// @raycast.argument1 { "type": "text", "placeholder": "First Date DEFAULT: now", "optional":true}
+// @raycast.argument2 { "type": "text", "placeholder": "Second Date" }
+// @raycast.description Given two dates returns the time between them in multiple units of mesure.
+// @raycast.packageName Developer Utils
```
Comment (unnamedd) [inline commands/developer-utils/time-between-dates.js:13]:
```suggestion
// @raycast.packageName Developer Utilities
```

File: commands/developer-utils/time-calculator.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Time Calculator
+// @raycast.mode compact
+
+// Optional parameters:
+// @raycast.icon 🕒
+// @raycast.argument1 { "type": "text", "placeholder": "Starting Date DEFAULT: now", "optional":true }
+// @raycast.argument2 { "type": "text", "placeholder": "Time to add or subtract eg: -44, 44, +44" }
+// @raycast.argument3 { "type": "text", "placeholder": "s, m, h, d, w, y DEFAULT: h", "optional":true}
```
Comment (unnamedd) [inline commands/developer-utils/time-calculator.js:12]:
```suggestion
// @raycast.argument3 { "type": "text", "placeholder": "s, m, h, d, w, y (Default: h)", "optional":true}
```

File: commands/developer-utils/time-calculator.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Time Calculator
+// @raycast.mode compact
+
+// Optional parameters:
+// @raycast.icon 🕒
+// @raycast.argument1 { "type": "text", "placeholder": "Starting Date DEFAULT: now", "optional":true }
```
Comment (unnamedd) [inline commands/developer-utils/time-calculator.js:10]:
```suggestion
// @raycast.argument1 { "type": "text", "placeholder": "Starting Date (Default: now)", "optional":true }
```

File: commands/developer-utils/time-calculator.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+// @raycast.argument2 { "type": "text", "placeholder": "Time to add or subtract eg: -44, 44, +44" }
```
Comment (unnamedd) [inline commands/developer-utils/time-calculator.js:11]:
```suggestion
// @raycast.argument2 { "type": "text", "placeholder": "Time to Add/Subtract (eg: -44, 44, +44)" }
```

File: commands/developer-utils/time-calculator.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+// @raycast.authorURL https://github.com/Mitra98t
+
+const dateReg = new RegExp(/^\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$/)
+
+var value = process.argv.slice(2)[1]
+var unit = process.argv.slice(2)[2] == "" ? "h" : process.argv.slice(2)[2]
+var operator = ""
+
+if (unit != "s" && unit != "m" && unit != "h" && unit != "d" && unit != "w" && unit != "y") {
+    console.log("Use s, m, h, d, w, y as units of mesure")
```
Comment (unnamedd) [inline commands/developer-utils/time-calculator.js:27]:
```suggestion
    console.log("Use s, m, h, d, w, y as units of measure")
```

File: commands/developer-utils/time-calculator.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+// @raycast.schemaVersion 1
+// @raycast.title Time Calculator
+// @raycast.mode compact
+
+// Optional parameters:
+// @raycast.icon 🕒
+// @raycast.argument1 { "type": "text", "placeholder": "Starting Date DEFAULT: now", "optional":true }
+// @raycast.argument2 { "type": "text", "placeholder": "Time to add or subtract eg: -44, 44, +44" }
+// @raycast.argument3 { "type": "text", "placeholder": "s, m, h, d, w, y DEFAULT: h", "optional":true}
+// @raycast.description Add and Subtract specified amount of time from given date.
```
Comment (unnamedd) [inline commands/developer-utils/time-calculator.js:13]:
```suggestion
// @raycast.description Add or Subtract specified amount of time from given date.
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hello @Mitra98t, 
Nice to have your first pull request in our repository!

Please, take a look into my comments and let me know when you finish to address them!

Welcome aboard!

**Comment by Mitra98t** [comment]
Hello @unnamedd 
I should have resolved every comment with the last commit.
Hope I've done well!

**Review by unnamedd — APPROVED** [review]
Hello @Mitra98t,

it looks good to me! Thanks for your changes! 
Your PR should be merged soon!

**Comment by RSO** [comment]
Thank you for the contribution, and welcome aboard!

**Pull request merged**